Runtime/Interaction/SnapZone.cs

```diff
@@ -104,13 +129,41 @@ public Material ValidationMaterial
         /// </summary>
         protected XRBaseInteractable ForceSelectTarget { get; set; }
 
-        private Material previewMaterial;
+        protected Vector3 attachOffset = Vector3.zero;
+        
         private Mesh previewMesh;
+        
+        [HideInInspector]
```

Comment:
If it is internal there is no need for `[HideInInspector]`

Reply:
removed